controllers/postsController.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../models");
+
+// PostsController methods
+module.exports = {
+
+  findAll: (req, res) => {
+    db.Post
+      .find(req.query)
```

Comment:
I'm not sure that you need `req.query` here if you're finding all posts

Reply:
not sure what I'd be passing into .find then.  Will look at this and hopefully have a revision. Otherwise we can confer tomorrow.

Reply:
When you want to find all with no filler you just leave it as `.find()`. You pass nothing into it.

Reply:
Roger that.  Thanks man.